src/Directory.Packages.props

```diff
@@ -55,5 +55,6 @@
   <ItemGroup>
     <PackageVersion Include="System.Net.Http" Version="4.3.4" />
     <PackageVersion Include="System.Text.RegularExpressions" Version="4.3.1" />
+    <PackageVersion Include="System.Private.Uri" Version="4.3.2" />
```

Comment:
Hmm, I removed this a few days ago thinking it would be OK since I have nothing that explicitly references it, as far as I can tell.